content/kubernetes/re-clusters/delete_custom_resources.md

```diff
@@ -0,0 +1,74 @@
+---
+Title: Delete custom resources
+linkTitle: Delete custom resources
+description: This article explains how to delete Redis Enterprise clusters and Redis Enterprise databases from your Kubernetes environment.
+weight: 90
+alwaysopen: false
+categories: ["Platforms"]
+aliases: [
+    /kubernetes/re-clusters/delete_custom_resources.md,
+    /kubernetes/re-clusters/delete_custom_resources/,
+]
+---
+
+## Delete a database
+
+To delete a database managed by the Redis Enterprise Kubernetes operator, run `kubectl delete redb <your-db-name>` from your K8s cluster.
+
+When you delete a database, your data and the REDB custom resource file are also deleted.
```

Comment:
This phrasing might confuse users to think that the actual redb.yaml file used to created the REDB is deleted too, which isn't the case. So, I'd avoid using "file" here and just say "the REDB custom resource are also deleted".